ui/src/components/easter/invitation/InvitationContainer.js

```diff
@@ -1,13 +1,63 @@
-import { Text } from "@chakra-ui/react";
+import * as React from 'react';
+import { useMediaQuery } from '@chakra-ui/media-query';
+import { Center, Image, Stack, Text, Flex, Link, extendTheme } from '@chakra-ui/react';
+import Invitation1 from './blie.png';
+import Invitation2 from './red.png';
 
 const InvitationContainer = () => {
+  
+  const [isNotSmallerScreen] = useMediaQuery("(min-width:600px)");
+  console.log(isNotSmallerScreen);
   return (
-    <>
-      <Text textStyle='Didot'>
-        Inivitation
-      </Text>
-    </>
-  )
+    <Center maxW="auto" mx="auto">
+      <Stack spacing={1}>
+        <Text 
+          textStyle='NextSoutherlandSerif'
+          textAlign='center'
+          fontSize='32px'>
```

Comment:
can we change this to '18px' when `isNotSmallerScreen` is true per the figma design?
also same comments for the other Text elements below